readers/openapi.yml

```diff
@@ -123,6 +125,21 @@ components:
         default: 0
         minimum: 0
       required: false
+    Publisher:
+      name: Publisher
+      description: Unique thing identifier.
+      in: query
+      schema:
+        type: string
+        format: uuid
+      required: false
+    Name:
+      name: name
+      description: SenML message name.
+      in: query
+      schema:
+        type: string
```

Comment:
"OpenAPI 3 introduces support for objects as query parameters." Please see here https://www.baeldung.com/openapi-json-query-parameters, specifically here https://www.baeldung.com/openapi-json-query-parameters#OAP3

Reply:
@darkodraskovic Looks like this explains more how to generate an API that accept JSON objects. We don't support this in our services. As you previously did I would follow: https://swagger.io/docs/specification/describing-parameters/

Reply:
This also allows for automatic Postman collections etc. I'm not sure, but it seams like a standard way to specify json string in query. Just add
```
        schema:
          type: object
```
And see if the editor complains.